fastapi_cache/decorator.py

```diff
@@ -92,8 +95,11 @@ async def ensure_async_func(*args: P.args, **kwargs: P.kwargs) -> R:
                     # see above why we have to await even although caller also awaits.
                     return await run_in_threadpool(func, *args, **kwargs)
 
-
             copy_kwargs = kwargs.copy()
+            if key_builder_exclude_field:
+                for key in kwargs.keys():
+                    if key in key_builder_exclude_field:
+                        copy_kwargs.pop(key)
```

Comment:
This might be better.
```python
if key_builder_exclude_field:
    for key in key_builder_exclude_field:
        copy_kwargs.pop(key, None)
```

Reply:
Yes, this is be better.

Reply:
would it be better to use the value and check for instance of Session rather than use the key name?

I used this in a custom key handler to remove any SqlAlchemy sessions:

`filtered_kwargs = {k: v for k, v in kwargs.items() if not isinstance(v, Session)}`

Reply:
You also need to introduce a heavy dependency and not a universal solution.